assembly/cli/src/site/markdown/index.md.vm

```diff
@@ -110,6 +110,12 @@ SECRET=password12345 java -jar path\to\ ${docName}.jar path\to\toolConfiguration
 Should you execute it within the command, the variable will not be saved in your environment but only exist for the run of the command.
 ${docNameCap} adheres to standards and only renders environment variables that are written in upper case letters.
 
+Alternatively, it is possible to specify environment variables inside a properties file which can be given to the analyzer command:
```

Comment:
Maybe you could mention that in the properties file also non-uppercase variables are taken, if I am not mistaken.

Reply:
Yes you are right, I checked this.